tests/functional/conftest.py

```diff
@@ -8,15 +8,16 @@
 from webtest import TestApp
 
 from h._compat import text_type
-from tests.common.fixtures import es_client  # noqa: F401
+from tests.common.fixtures import es6_client  # noqa: F401
 from tests.common.fixtures import init_elasticsearch  # noqa: F401
-from tests.common.fixtures import delete_all_elasticsearch_documents  # noqa: F401
 from tests.common.fixtures.elasticsearch import ELASTICSEARCH_HOST
```

Comment:
Are `ELASTICSEARCH_HOST` / `es.host` still needed?

Reply:
They are until we remove the get_client/ request.es.